proto/api.proto

```diff
@@ -45,6 +45,23 @@ message ClientCmdID {
   optional int64 random = 2 [(gogoproto.nullable) = false];
 }
 
+// ReadConsistencyType specifies what type of consistency is observed
+// during read operations.
+enum ReadConsistencyType {
+  option (gogoproto.goproto_enum_prefix) = false;
+  // CONSISTENT reads are guaranteed to read committed data; the
+  // mechanism relies on clocks to determine lease expirations.
+  CONSISTENT = 0;
+  // CONSENSUS requires that reads must achieve consensus. This is a
+  // stronger guarantee of consistency than CONSISTENT.
+  //
+  // TODO(spencer): current unimplemented.
+  CONSENSUS = 1;
+  // INCONSISTENT reads are not guaranteed to read committed data,
```

Comment:
The comment technically doesn't sound like "read uncommitted", but we could flesh it out a little bit:

```
// INCONSISTENT reads return the latest available, committed value. They are more efficient but may occasionally read stale values since open intents are ignored.
```